### PR TITLE
Comment cleanup

### PR DESCRIPTION
**Pull request by brian-intel**

### CODE REVIEW

File: as-vending/routes/controller.go [hunk 1]
```diff
@@ -221,6 +221,6 @@ func (c *Controller) BoardStatus(writer http.ResponseWriter, req *http.Request)
 
 	_, writeErr := writer.Write([]byte(returnval))
 	if writeErr != nil {
-		fmt.Printf("Failed to write item data back to caller\n")
+		c.lc.Error("Failed to write item data back to caller\n")
```
Comment (jim-wang-intel) [inline as-vending/routes/controller.go:224]: remove \n

File: as-vending/routes/controller.go [hunk 2]
```diff
@@ -111,13 +111,13 @@ func (c *Controller) BoardStatus(writer http.ResponseWriter, req *http.Request)
 	body := make([]byte, req.ContentLength)
 	_, err := io.ReadFull(req.Body, body)
 	if err != nil {
-		fmt.Printf("Failed to read request data\n")
+		c.lc.Errorf("Failed to read request data\n")
```
Comment (jim-wang-intel) [inline as-vending/routes/controller.go:114]: remove \n

File: as-vending/routes/controller.go [hunk 3]
```diff
@@ -111,13 +111,13 @@ func (c *Controller) BoardStatus(writer http.ResponseWriter, req *http.Request)
...
 	}
 
 	// Unmarshal the string contents of request into a proper structure
 	var boardStatus functions.ControllerBoardStatus
 	if err := json.Unmarshal(body, &boardStatus); err != nil {
-		fmt.Printf("Failed to read request data\n")
+		c.lc.Errorf("Failed to read request data\n")
```
Comment (jim-wang-intel) [inline as-vending/routes/controller.go:120]: ditto

File: as-vending/routes/controller.go [hunk 4]
```diff
@@ -84,20 +84,20 @@ func (c *Controller) ResetDoorLock(writer http.ResponseWriter, req *http.Request
...
+	c.lc.Debugf("Inference: %t", c.vendingState.InferenceDataReceived)
+	c.lc.Debugf("door: %t", c.vendingState.DoorClosed)
 
 	// Write the HTTP status header
 	writer.WriteHeader(http.StatusOK)
 
 	_, writeErr := writer.Write([]byte(returnval))
 	if writeErr != nil {
-		fmt.Printf("Failed to write item data back to caller\n")
+		c.lc.Errorf("Failed to write item data back to caller\n")
```
Comment (jim-wang-intel) [inline as-vending/routes/controller.go:100]: remove \n

**Review by jim-wang-intel — APPROVED** [review]
LGTM

**Review by joshua-silverio — APPROVED** [review]
lgtm

**Review by vli11 — APPROVED** [review]
lgtm

**Pull request merged**